changelog/_unreleased/2021-02-15-remove-entity-caching-in-dbal-entity-hydrator.md

```diff
@@ -0,0 +1,11 @@
+---
+title: remove entity caching in DBAL Entity Hydrator
+issue: NEXT-13735
+author: Alexander Bachmann
+author_email: email.bachmann@gmail.com
+author_github: AlexBachmann
+---
+# Core
+* Removed entity caching in DBAL Entity Hydrator to avoid
```

Comment:
I think something went wrong with your changelog here. It should only be single line not multi line. Would be great if you could change that. Besides that the PR looks good 🙂

Reply:
I inlined the entry